install/install.sql

```diff
@@ -726,3 +735,26 @@ CREATE TABLE IF NOT EXISTS `v3_ipv4_mapping` (
     COLLATE = utf8mb4_unicode_ci;
 
 -- --------------------------------------------------------------------------------
+--
+-- Table structure for player ranking scores `v3_rankings`
+--
+CREATE TABLE IF NOT EXISTS `v3_rankings` (
+    `id`             INT UNSIGNED NOT NULL,
+    `username`       VARCHAR(30) CHARACTER SET ascii NOT NULL,
+    `scores`         DOUBLE NOT NULL,
```

Comment:
What is the difference between `scores` and `max_scores`?

Reply:
https://github.com/supertuxkart/stk-code/compare/master...STK-helper:RankingFormula

Reply:
Scores corresponds to the current ranking points : how much does the player have right now.
Max_scores corresponds to the maximum attained ranking points : what is the highest amount the player got.

Storing this data has two purposes : 
- It is used in the computation formula to slow down the points exchange of people having a max ranking over several ceilings. This is needed so that average players can have their points evolve quick enough while it is hard to do enough good performances to get at the top (avoid someone having some luck early and achieving too easily a high ranking he'll then lose quickly). Also, players at the top will often do more races (it generally match more invested players), so again a slower change rate for them is better. Using the max achieved is a better measure for this than the current ranking (that's how the FIDE does it for its international chess ranking).
- It is a stat players will be interested in. Losing some current ranking points will be less frustrating as long as they still have their maximum achieved points showing what they achieved.